routes/api.js

```diff
@@ -255,6 +259,8 @@ router.post('/user-access', function(req, res, next) {
         'Content-Type': 'text/plain',
         'station-state': 'disabled',
       });
+      db.logEvent('log off station', + req.headers['stationd-id'] + 'by user' + res.locals.obj.user).then(function(){
```

Comment:
Same as the log on logEvent.  This log needs to be edited to include proper spacing and the same issues with req.headers['stationd-id'] and res.locals.obj.user seem to not be working as you had assumed.